### DISCUSSION TITLE
Nginx 설정의 잦은 변경

### DISCUSSION THREAD

**Discussion by LuizyHub** [woowa-techcamp-2024/Team5-Guys#68] [General]
Nginx 의 설정도 자주 변경되게 느껴집니다!
이것도 형상관리를 통해서 변경사항을 Git으로 관리하고 싶은데 생각이 궁금합니다

### 지난 변경 이력
- api.logbat.info 추가
  - ssl 인증서 추가
  - 프록시 설정 추가
- sdk.logbat.info 추가
  - ssl 인증서 추가
  - 프록시 설정 추가
- api.logbat.info/logs CORS 설정 추가

### 현재 변경될 이력
- view.logbat.info 추가
  - ssl 인증서 추가
  - 프록시 설정 추가
- lua 스크립트 추가
- Count API 추가

### 앞으로 변경될 수 있는 내용
- api.logbat.info/logs CORS Origin 화이트 리스트 추가

**Comment by LuizyHub** [comment]
@tidavid1 @miiiinju1 
리뷰어 기능이 없어서 댓글에 추가합니다!

**Comment by kyungminnnnn** [comment]
변경 부분 진행 완료 이후 문서화 링크 포함해서 댓글로 링킹해주는 것도 좋을 것 같습니다 :D

**Reply by LuizyHub** [reply]
현재 그 방식으로 매번 개발 위키에 새로운 이력을 추가하고 있는데, 이게 변경 이력 파악도 어렵고 매번 문서화 하는것이 불편하게 느껴집니다,,,

**Reply by kyungminnnnn** [reply]
내일 스크럼때 의견 공유 하면 좋을 것 같아여

**Comment by LuizyHub** [comment]
한번 실험적으로 도입해보고 PR 만들어볼게요!
